starcheck/utils.py

```diff
@@ -103,8 +105,74 @@ def get_data_dir():
     return sc_data if os.path.exists(sc_data) else ""
 
 
-def make_ir_check_report(**kwargs):
-    return ir_zone_ok(**kwargs)
+class AlternatingPixelModeSelectTransition(FixedTransition):
+    command_attributes = {'tlmsid': 'AAC1CCSC', 'hex': '348E040'}
+    state_keys = ['pixel_mode']
+    transition_key = ['pixel_mode']
+    transition_val = ['ALT']
+
+
+class OriginalPixelModeSelectTransition(FixedTransition):
+    command_attributes = {'tlmsid': 'AAC1CCSC', 'hex': '3492040'}
+    state_keys = ['pixel_mode']
+    transition_key = ['pixel_mode']
+    transition_val = ['ORIG']
+
+
+def get_pixel_mode_kadi_continuity(backstop_file):
+    rltt, sched_stop = get_rltt_and_sched_stop(backstop_file)
+    if rltt.date < '2023:178':
```

Comment:
I didn't see a great way to add a default or date-based continuity for the new FixedTransitions I defined, so I just made a new method to hard-code it for this.